app/controllers/api/v1x0/actions_controller.rb

```diff
@@ -5,7 +5,7 @@ class ActionsController < ApplicationController
 
       def index
         req = Request.find(params.require(:request_id))
-        authorize req
+        authorize req, :show?
 
```

Comment:
This `authorize` check will be moved to `ActionPolicy` in that user capabilities PR

Reply:
Understand, it is done by your PR.